crates/node/src/cli.rs

```diff
@@ -126,6 +126,14 @@ pub struct Config {
 
     #[arg(long, long_help = "GPU PCI devices", env = "GEVULOT_GPU_DEVICES")]
     pub gpu_devices: Option<String>,
+
+    #[arg(
+        long,
+        long_help = "Watchdog listen address",
+        env = "GEVULOT_WATCHDOG_LISTEN_ADDR",
+        default_value = "127.0.0.1:8888"
+    )]
+    pub http_watchdog_listen_addr: SocketAddr,
```

Comment:
```suggestion
        long_help = "Watchdog listen address",
        env = "GEVULOT_WATCHDOG_LISTEN_ADDR",
        default_value = "127.0.0.1:8888"
    )]
    pub http_watchdog_listen_addr: SocketAddr,
```
I'd maybe name this as a healthcheck for the end user. The end user doesn't care how it has been implemented, but rather what it can provide. Healthcheck is common name for an HTTP interface that provides liveness status from `/healthz`.
```suggestion
        long_help = "Healthcheck listen address",
        env = "GEVULOT_HEALTHCHECK_LISTEN_ADDR",
        default_value = "127.0.0.1:8888"
    )]
    pub http_healthcheck_listen_addr: SocketAddr,
```